apps/web/src/utils/initCCA.ts

```diff
@@ -44,6 +49,31 @@ const initCCA = (
     initNextJsTrackPageview({
       nextJsRouter: router,
     });
+
+    Experiment.initialize(ampDeploymentKey, {
```

Comment:
Might as well just not do this here anymore.

Reply:
Makes sense, removed.